libs/api/__tests__/specs/mediaAdapter.test.ts

```diff
@@ -79,7 +79,7 @@ describe('Karma Media Adapter', () => {
           filename: 'test.jpg',
           mimetype: 'image/jpeg',
           encoding: 'fakeEncoding',
-          createReadStream: () => imageStream
+          createReadStream: () => imageStream as any
```

Comment:
imageStream is `fs.ReadStream` but `createReadStream` expects a `fs-capactiro.ReadStream` which does not seem to be compatible.

Dunno how this test works on master